runeq/resources/stream_metadata.py

```diff
@@ -758,39 +758,46 @@ def get_stream_metadata(
     else:
         stream_ids = list(stream_ids)
 
-    result = client.execute(
-        statement=query,
-        stream_ids=stream_ids,
-    )
+    # Query stream list in batches of <= 100 streams, since graph api cannot
+    # query for more than 100 streams at once.
+    stream_list_results = []
+    for i in range((len(stream_ids) // 100) + 1):
+        result = client.execute(
+            statement=query,
+            stream_ids=stream_ids[i * 100: (i + 1) * 100],
```

Comment:
Totally optional, but another pattern (that I find a little easier to read) is to have `range` skip by 100 numbers:

```suggestion
    for start in range(0, len(stream_ids), 100):
        result = client.execute(
            statement=query,
            stream_ids=stream_ids[start:start + 100],
```

(Python doesn't care if your upper index is out of bounds)